MetaMask/metamask-extension
# adding Filecoin logo for the MetaMask UI

## Explanation
Filecoin virtual Machain is EVM-compatible and there are a lot of devs and users who used MetaMask for FIL token transfer or smart contract interaction on Filecoin network.
Users are able to add the Filecoin network to MetaMask and use all the features. But the Filecoin logo is not showing up anywhere in the UI. So we would love to add the Filecoin logo to the proper place on MetaMask UI.

## Screenshots/Screencaps

This PR is mainly the UI work, adding Filecoin logo to proper place.

### Before

<img width="371" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/34025001/2b84d1da-7287-4219-8a22-941a3eebd741">

<img width="371" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/34025001/ede038f8-460f-468b-a20a-d9ce64fbc7c9">


### After

<img width="371" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/34025001/5d8cc237-0ef0-44c3-aaf5-0401187c75e1">

<img width="371" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/34025001/3395e1f9-83fe-486d-b0dc-53378ebf02f5">


## Manual Testing Steps

- add Filecoin to the Network on MetaMask
- switch to Filecoin network
- check if the logo shows up on network display as well as token display
- try to transfer FIL token and verify if Filecoin logo is displayed.

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
